pom.xml

```diff
@@ -42,25 +69,11 @@
     <plugin.osmaven.version>1.6.2</plugin.osmaven.version>
     <plugin.signature.version>1.1</plugin.signature.version>
     <plugin.surfire.version>3.0.0-M3</plugin.surfire.version>
-    <!-- keep below 3.1.0 because 3.1.0 fails on java 8. -->
-    <plugin.javadoc.version>3.0.1</plugin.javadoc.version>
+    <plugin.javadoc.version>3.1.1</plugin.javadoc.version>
     <plugin.source.version>3.0.1</plugin.source.version>
     <plugin.compiler.version>3.8.0</plugin.compiler.version>
   </properties>
 
-  <organization>
-    <name>Java Native</name>
-    <url>https://github.com/java-native/</url>
-  </organization>
```

Comment:
I allowed myself to remove the organization tag, as it is not mandatory, and does not really represents an organization.